ngi_pipeline/engines/piper_ngi/local_process_tracking.py

```diff
@@ -73,12 +98,7 @@ def update_charon_with_local_jobs_status(quiet=False, config=None, config_file_p
                                                                    e))
                 LOG.error(error_text)
                 if not config.get('quiet'):
-                    mail_analysis(project_name=project_name,
-                                  sample_name=sample_id,
-                                  engine_name=engine,
-                                  level="ERROR",
-                                  info_text=error_text,
-                                  workflow=workflow)
+                    mail_ngi_log_message(sample_env,"ERROR",error_text)
```

Comment:
We can use that style if we prefer that, for me it makes the code even more cluttered. It is already a rather long wall of text, and the logic can not be seen clearly when it is peppered with long list of arguments.